pkg/docker/docker.go

```diff
@@ -347,16 +347,26 @@ func (d *stiDocker) GetOnBuild(name string) ([]string, error) {
 // and returns the image metadata
 func (d *stiDocker) CheckAndPullImage(name string) (*docker.Image, error) {
 	name = getImageName(name)
+	displayName := name
+
+	if glog.Is(3) == false {
+		// Summarize name for lower level log output
+		split := strings.Split(name, "@")
+		if len(split) > 1 && len(split[1]) > 18 {
+			displayName = split[0] + "@" + split[1][:18] + "..."
```

Comment:
@csrwng  @bparees  Clayton suggested a "summary" at this logging level. The choice of 18 was somewhat arbitrary: I wanted the user to see enough of the sha for it to be unlikely to correspond to existing images on the system, but shorter than the 80 chars Clayton believed was problematic.
So it changes:
**Image "centos/php-56-centos7@sha256:51c3e2b08bd9fadefccd6ec42288680d6d7f861bdbfbd2d8d24960621e4e27f5" not available locally, pulling ...**
to 
**Image "centos/php-56-centos7@sha256:51c3e2b08bd..." not available locally, pulling ...**